### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+    if files_names[1] == files_names[2]:
```
Comment (Dimosphen1) [inline app/main.py:3]: You should additionally check that the `cp` word is present in a command and it is placed in the first place. Additionally, there is a need to check that the input command contains exactly three elements according [to the checklist](https://github.com/AnyoneClown/py-copy-file/blob/develop/checklist.md#code-efficiency). Also, don't use indexes to retrieve elements from the split command, prefer unpacking and writing values to variables after the length check.

File: app/main.py [hunk 2]
```diff
@@ -1 +1,9 @@
...
+        return
+
+    with (open(files_names[1], "r") as main_file,
+          open(files_names[2], "w") as copied_file):
+        for line in main_file:
+            copied_file.write(line)
```
Comment (Dimosphen1) [inline app/main.py:9]: Do not iterate through lines here, you could use `.writelines()` or a combination of `.write()` + `.read()` methods.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested,

File: app/main.py [hunk 3]
```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+
+    cp_command = files_names[0]
+    first_file = files_names[1]
+    second_file = files_names[2]
+
```
Comment (viktoria-rybenchuk) [inline app/main.py:7]: Extract the command and file names using unpacking. But first check that length command.split is 3

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**